tests/fast_reboot_dump_dbs/APPL_DB.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
should you change the file name to be with lowercase letters instead of capital letters? or there is a reason for having it like that?

Reply:
I am using this string as capital for other things in the code as well, here for example:
db = getattr(dbconn, db_dump_filename.replace('.json',''))